Models path adjustment in ONNX tests

tomdol opened the pull request:
### Details:
 - ONNX tests modified to use another path to the models
 - This change lets the test binaries operating on various models be executed outside of the bin/intel64/Debug (or Release) directory

### Tickets:
 - 88917

ilya-lavrenov commented:
But in this case the binaries are not relocatable, i.e. I cannot copy it on different machine and run the tests

tomdol commented:
> But in this case the binaries are not relocatable, i.e. I cannot copy it on different machine and run the tests

I've reverted the previous changes and updated all ONNX tests. Now they can be executed out of the binary's directory.
